docs/contribute/modules/index.md

```diff
@@ -27,9 +27,9 @@ After setting up your repository and linking it to your GitHub account, you can
 You may implement your code in the ``init.py`` located in the respective feature folder in ``src/autora``.
```

Comment:
This line looks to be formatted incorrectly – it should show `__init__.py` not `init.py`
```
You may implement your code in the ``__init__.py`` located in the respective feature folder in ``src/autora``.
```

Reply:
Thanks for the catch! I committed a fix implementing the suggested renaming from 'init.py' to '__init__.py'